src/IntegralArrays.jl

```diff
@@ -1,5 +1,78 @@
 module IntegralArrays
 
 # package code goes here
+using Images, ClosedIntervals
```

Comment:
Hmm...for Integral Arrays, I dont think anyone would index with a reverse ordered interval. Maybe we can just use Tuples instead?